datadistribution.sh

```diff
@@ -0,0 +1,67 @@
+package: DataDistribution
+version: "%(tag_basename)s"
+tag: v0.5.1
+requires:
+  - boost
+  - FairLogger
+  - FairMQ
+  - Monitoring
+  - O2
+build_requires:
+  - CMake
+  - "GCC-Toolchain:(?!osx)"
+  - ms_gsl
+  - CodingGuidelines
```

Comment:
Why do you need the CodingGuidelines?

Reply:
leftover from template copy/paste...
Removed